### ISSUE TITLE
Not working as described

### ISSUE THREAD

**Issue by siffegh** [Josua996/AutoTag#2]
##  Expected Behavior
1. Clicking the Autotag button doesn't produce the list of tags.
2. The Options page shows only three lines, all of which say 'Missing string'

## Actual Behavior
1. Clicking the Autotag button should produce the list of tags.
2. The Options page should show more than that.

## Steps to Reproduce the Problem
Follow instructions to use the addon.
 
## Specifications

  - Thunderbird-Version: 91.4.0
  - Win7 Pro SP1

**Comment by Josua996** [comment]
First of all, thank you for your feedback! Unfortunately, I cannot reproduce the bug you describe. Therefore, I will test the addon tomorrow on other devices. 
Did you install the addon from the Thunderbird website, the GitHub repository or directly from code? 

You could help by sending me the error logs from the developer console. Go to _Tool->Developer Tools->Debug Add-ons_ and select "Inspect" in the list for the "Auto Tag" entry. A new tab will open, there you can select "Console" at the top. It would be great if you could upload a screenshot or copy the messages.

If I can reproduce the error tomorrow on another device, I'll let you know. Thanks for your help and have a nice evening.

**Comment by siffegh** [comment]
I installed if from the TB addons web site. Here are the errors.
![Screenshot_1](https://user-images.githubusercontent.com/18728013/146102741-548d79bc-709e-4037-b80c-39ccec941d30.jpg)

**Comment by Josua996** [comment]
Thanks for the quick reply and the screenshot! I was able to find the error because of the error logs.

The addon tries to adapt its colors to the active theme. But since your theme doesn't provides colors via the theme API, the addon couldn't be initialized correctly.

As solution the default values of Thunderbird are used as fallback (Popup Dark/Settings Light).  In the next days I will add a setting where you can manually change the theme of the popup. I have updated the version on the TB website - depending on your settings the update should install automatically.

It would be great if you could give feedback if it works now :)

**Comment by siffegh** [comment]
Thanks. Tag-wise, it's functioning based on an email I sent to myself, but there are still a couple of glitches:

1. Visual - check boxes don't show the check marks
2. Visual - address overflow
3. Toolbar icon - not showing (log shows true  for that)

I'm using the System theme, but changing to Light or Dark didn't fix it. I also disabled userChrome/userContent just to make sure.

![Screenshot_1](https://user-images.githubusercontent.com/18728013/146117064-6f31aee8-b048-4825-a150-f5fb14c9739a.jpg)

**Comment by siffegh** [comment]
Thanks. In fact, my visual issues are because I have my colors (black text, gray background) set to always override colors specified by the content. I updated and tested with never override, and it looks fine.

#3 I'm referring to the Tag-Quickselect-Toolbar, which I presume to be different than the auto-tag icon. Can you post a screenshot of it?

**Comment by Josua996** [comment]
Are the ticks in the checkboxes now also displayed with your override rules? Otherwise I would look for a solution so that you can also use the addon with the override rules active. 
The Quickselect toolbar is displayed in the message above the content (screenshot). It may be that this is incompatible with other addons that change the appearance of the messages. 
<img width="824" alt="Quickselect" src="https://user-images.githubusercontent.com/16152662/146268481-807af1d6-80cb-40ca-87bc-a9e485b0c4f7.png">

**Comment by siffegh** [comment]
The ticks in the check boxes aren't displayed with my override rules.

I do see the he Quickselect toolbar; it's a bit confusing because you also refer to it as the auto-tag icon in the description.

**Comment by Josua996** [comment]
Sorry I accidentally closed the issue. To make the addon usable for you, styled checkboxes are now optional and have to be enabled in the settings. Since I now use system checkboxes I hope they work for you too.
I will revise the description. I am still dissatisfied with the solution of the Quickselect Toolbar anyway and will rewrite it completely when I find the time.  

I hope now the selection of the checkboxes works and is also displayed correctly. I wanted to thank you again for your feedback and your efforts! Without you I would never found these issues!

**Comment by siffegh** [comment]
Yes, I now see the checkmarks - thanks a lot! I'm happy to help; just let me know if you need more testing.

**Comment by siffegh** [comment]
BTW, I found a glitch on your README.md:

![image](https://user-images.githubusercontent.com/18728013/146308156-8fe84fd9-24d5-419b-9541-dbe0534c042a.png)

**Comment by Josua996** [comment]
I have corrected the link. 
If there is a certain function you can test or something like that I will let you know - thanks a lot. If you notice a bug or have an idea for a new feature you can always contact me or create an issue :)

Thanks again - you helped a lot! Have a nice day and have fun using the addon :)
